packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -460,6 +467,14 @@ export function getDestinationChainIds(chainId: ChainId): ChainId[] {
   const validDestinationChainIds =
     chains.find(chain => chain.chainID === chainId)?.partnerChainIDs || []
 
+  // add orbit-chains are also a part of the valid destination id's
+  const teleportAllowList = new Set(TELEPORT_ALLOWLIST[chainId] || [])
```

Comment:
Why do we need a Set here? `TELEPORT_ALLOWLIST[chainId]` should be unique?

Reply:
oh haha my bad :D I think I initially suggested it based on the code itself without checking the logic, but now looking it the second time I've noticed this

Reply:
Reverted the use of set